stdlib/public/core/CString.swift

```diff
@@ -400,13 +451,16 @@ extension String {
     return ("", false)
   }
 
-  /// Creates a string from the null-terminated sequence of bytes at the given
-  /// pointer.
+  /// Creates a new string by copying the null-terminated sequence of code units
+  /// referenced by the given pointer.
+  ///
+  /// If `nullTerminatedCodeUnits` contains ill-formed code unit sequences, this
+  /// initializer replaces them with the Unicode replacement character
+  /// (`"\u{FFFD}"`).
   ///
   /// - Parameters:
-  ///   - nullTerminatedCodeUnits: A pointer to a sequence of contiguous code
-  ///     units in the encoding specified in `sourceEncoding`, ending just
-  ///     before the first zero code unit.
+  ///   - nullTerminatedCodeUnits: A pointer to a null-terminated sequence of
+  ///     code units encoded in `sourceEncoding`.
   ///   - sourceEncoding: The encoding in which the code units should be
```

Comment:
Should this parameter label be changed to `encoding` as in all the other cases?

Reply:
Up to you! The only effect of that name change is in the documentation, so it's fine to update it. And while consistency is good, it's unlikely that people will be comparing these parameter names directly, since they don't appear on the method listing page.

Reply:
I think I'll do that label change. Thanks for the soundness check.